core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -0,0 +1,279 @@
+if($PSVersionTable.PSVersion.Major -lt 2) {
+    Write-Warning "This script requires PowerShell 2.0 or better; you have version $($Host.Version)."
+    return
+}
+
+$SCRIPTS_HOME = (Get-ChildItem $MyInvocation.MyCommand.Path).Directory.FullName
+$RESOLVED_JBOSS_HOME = (Get-ChildItem $MyInvocation.MyCommand.Path).Directory.Parent.FullName
+
+
+$global:SECMGR = $false
+$global:DEBUG_MODE=$false
+$global:DEBUG_PORT=8787
+$global:RUN_IN_BACKGROUND=$false
+$global:GC_LOG=$false
+#module opts that are passed to jboss modules
+$global:MODULE_OPTS = @()
+
+# A collection of functions that are used by the other scripts
+
+Function Set-Env {
+  $key = $args[0]
+  $value = $args[1]
+  Set-Content -Path env:$key -Value $value
+}
+
+Function Get-Env {
+  $key = $args[0]
+  if( Test-Path env:$key ) {
+    return (Get-ChildItem env:$key).Value
+  }
+  return $args[1]
+}
+
+Function Get-String {
+  $value = ''
+  foreach($k in $args) {
+    $value += $k
+  }
+  return $value
+}
+
+Function String-To-Array($value) {
+  $res = @()
+  if (!$value){
+  	return $res
+  }
+  $tmpArr = $value.split()
+  
+  foreach ($str in $tmpArr) {
+    if ($str) {
+	  $res += $str
+	}
+  }
+  return $res
+}
+
+Function Display-Environment { 
+$JAVA_OPTS = Get-Java-Opts
+# Display our environment
+Write-Host "================================================================================="
+Write-Host ""
+Write-Host "  WildFly Bootstrap Environment"
```

Comment:
I am tempted to change it everywhere with generic "Application Server" without the name....